doc/modules/grid_search.rst

```diff
@@ -1,10 +1,10 @@
-.. currentmodule:: sklearn.grid_search
+.. currentmodule:: sklearn.model_selection
 
 .. _grid_search:
 
-===============================================
-Grid Search: Searching for estimator parameters
-===============================================
+==========================================
+Search: Searching for estimator parameters
+==========================================
 
 Parameters that are not directly learnt within estimators can be set by
 searching a parameter space for the best :ref:`cross_validation` score.
```

Comment:
Hum there are some issues with this section of the docs but I guess this is not the PR for that.